### ISSUE TITLE
When does the training stop?

### ISSUE THREAD

**Issue by qmin2** [HKUNLP/instructor-embedding#41]
Hello,

Was there a specific reason for ending the training at 40K steps?
What criteria or basis(i.e. metric) did you use to determine the stopping point?

**Comment by hongjin-su** [comment]
Hi, Thanks a lot for your interest in the INSTRUCTOR model!

To determine the stopping point of the training, you may observe the loss curve. When the loss decreases slowly and the curve becomes flat, you may stop the training process.

Feel free to add any further questions or comments!

**Comment by qmin2** [comment]
Thanks a lot !

**Comment by ashokrajab** [comment]
Hi @hongjin-su ,
Just wanted to know whether there is any default support for logging the training loss?
I do not see any log files being generated that captures the training losses.
